client/stylesheets/layouts/_header.scss

```diff
@@ -1,14 +1,12 @@
 [role='banner'] {
-  $background: $dark-grey !default;
-
   @include media-tablet-up {
     min-height: modular-scale(3);
     padding: modular-scale(-1) modular-scale(1);
   }
 
   @include clearfix;
-  background-color: $background;
-  border-bottom: 1px solid darken($background, 10);
+  background-color: $background-dark;
```

Comment:
You might want to use `background` instead of `background-color` for consistency